activation/activation.go

```diff
@@ -875,11 +891,18 @@ func (b *Builder) searchPositioningAtx(
 		VerifyChainOpts.AssumeValidBefore(time.Now().Add(-b.postValidityDelay)),
 		VerifyChainOpts.WithTrustedID(nodeID),
 		VerifyChainOpts.WithLogger(b.logger),
+		VerifyChainOpts.PrioritizeCall(),
 	)
 	if err != nil {
-		logger.Info("search failed - using golden atx as positioning atx", zap.Error(err))
-		id = b.conf.GoldenATXID
+		if previous != nil {
```

Comment:
since this is the only block inside the `if` clause, you could reduce nesting by just checking `if err != nil && previous != nil`

Reply:
No, I can't, because we set up prev id or golden id only in case if err != nil. 
So if I set `f err != nil && previous != nil` and for example, `err == nil` found id will be replaced with golden atx id. And it is wrong